installer/templates/phx_web/views/error_helpers.ex

```diff
@@ -1,19 +1,19 @@
 defmodule <%= web_namespace %>.ErrorHelpers do
   @moduledoc """
   Conveniences for translating and building error messages.
-  """
-<%= if html do %>
+  """<% error_message = if gettext, do: "translate_error(error)", else: "error" %><%= if html do %>
+
   use Phoenix.HTML
 
   @doc """
   Generates tag for inlined form input errors.
   """
   def error_tag(form, field) do
     Enum.map(Keyword.get_values(form.errors, field), fn error ->
-      content_tag(:span, translate_error(error), class: "help-block")
+      content_tag(:span, <%= error_message %>, class: "help-block")
```

Comment:
I think in case there's no gettext there needs to be an alternative implementation of error message interpolation - the code in this PR will actually crash when executed, trying to put the error tuple inside of a `span` tag.

Reply:
Ah, yes, good catch. We should probably keep the `translate_error` function but implement it differently. It would be something like this:

    def translate_error({msg, opts}) do
      Enum.reduce(opts, msg, fn {key, value}, acc ->
        String.replace(acc, "%{#{key}", to_string(value))
      end)
    end

Reply:
Added the alternative implementation here: https://github.com/phoenixframework/phoenix/pull/3387/commits/982d86dd4befecad3e2daeb412ffa9188cccf15e